gulpfile.js

```diff
@@ -173,80 +171,74 @@ gulp.task('deploy:site:prod', 'deploy to production site', function() {
 
 gulp.task('deploy:api:prod', 'deploy to production api app engine', function() {
   return run(
-    'cd api && goapp deploy -application  amp-by-example-api -version 1')
-    .exec();
+      'cd api && goapp deploy -application  amp-by-example-api -version 1')
+      .exec();
 });
 
 gulp.task('deploy:site:staging', 'deploy to staging app engine', function() {
   return run('goapp deploy -application ' + config.appId + ' -version 1')
-    .exec();
+      .exec();
 });
 
 gulp.task('copy:images', 'copy example images', function() {
   return gulp.src(paths.images)
-    .pipe(cache('img'))
-    .pipe(gulp.dest(paths.dist.img));
+      .pipe(cache('img'))
+      .pipe(gulp.dest(paths.dist.img));
 });
 
 gulp.task('copy:videos', 'copy example videos', function() {
   return gulp.src(paths.videos)
-    .pipe(cache('video'))
-    .pipe(gulp.dest(paths.dist.video));
+      .pipe(cache('video'))
+      .pipe(gulp.dest(paths.dist.video));
 });
 
 gulp.task('copy:json', 'copy example json', function() {
   return gulp.src(paths.json)
-    .pipe(cache('json'))
-    .pipe(gulp.dest(paths.dist.json));
+      .pipe(cache('json'))
+      .pipe(gulp.dest(paths.dist.json));
 });
 
 gulp.task('copy:css', 'copy css', function() {
   return gulp.src(paths.css)
-    .pipe(cache('css'))
-    .pipe(gulp.dest(paths.dist.css));
+      .pipe(cache('css'))
+      .pipe(gulp.dest(paths.dist.css));
 });
 
 gulp.task('copy:node-modules', function() {
   const modules = paths.nodeModules.map(
-    module => 'node_modules/' + module + '/**/*'
+      module => 'node_modules/' + module + '/**/*'
   );
   return gulp.src(modules, {
-    base: 'node_modules'
+    base: 'node_modules',
   }).pipe(gulp.dest(paths.dist.dir));
 });
 
 gulp.task('copy:fonts', 'copy example fonts', function() {
   return gulp.src(paths.fonts)
-    .pipe(cache('fonts'))
-    .pipe(gulp.dest(paths.dist.fonts));
+      .pipe(cache('fonts'))
+      .pipe(gulp.dest(paths.dist.fonts));
 });
 
 gulp.task('copy:well-known', 'copy well-known folder', function() {
   return gulp.src(paths.wellknown)
-    .pipe(cache('wellknown'))
-    .pipe(gulp.dest(paths.dist.wellknown));
-});
-
-gulp.task('copy:scripts', 'copy scripts', function() {
-  return gulp.src(paths.scripts)
-    .pipe(cache('scripts'))
-    .pipe(gulp.dest(paths.dist.scripts));
+      .pipe(cache('wellknown'))
+      .pipe(gulp.dest(paths.dist.wellknown));
 });
 
 gulp.task('copy:license', 'copy license', function() {
   return gulp.src('LICENSE')
-    .pipe(cache('license'))
-    .pipe(rename(function(path) {
-      path.extname = ".txt";
-    }))
-    .pipe(gulp.dest(paths.dist.dir));
+      .pipe(cache('license'))
+      .pipe(rename(function(path) {
```

Comment:
Nit: maybe replace this kind of thing with arrow functions (there's a few other places in the file as well)?

Reply:
We haven't used these anywhere yet.